README.md

```diff
@@ -1,61 +1,98 @@
-  - Implementations:
-    - [ ] Bucket-brigade (original circuit model, and updated constant-depth model)
-    - [ ] Various qROMs
-    - [ ] Quantum state preparation ("select/SWAP" oracles)
-    - [ ] Application-specific qROMs
-  - Samples:
-    - [ ] Instantiation and querying all implemented qRAMs/qROMs
-    - [ ] Resource estimation
-- Written report about the different methods used
+[![q# code style](https://img.shields.io/badge/code%20style-Q%23-blue)](https://docs.microsoft.com/en-us/quantum/contributing/style-guide?tabs=guidance)
+[![q# APIcode style](https://img.shields.io/badge/code%20style-Q%23%20API-ff69b4)](https://docs.microsoft.com/en-us/quantum/contributing/style-guide?tabs=guidance)
+[![c# APIcode style](https://img.shields.io/badge/code%20style-C%23-lightgrey)](https://docs.microsoft.com/dotnet/csharp/programming-guide/inside-a-program/coding-conventions)
+[![CoC](https://img.shields.io/badge/code%20of%20conduct-contributor%20covenant-yellow)](CODE_OF_CONDUCT.md)
 
-### Pre-reading
+## Screenshots
 
-- Learn about how classical RAMs work (bitlines, wordlines, capacitors, structure, etc.)
+**Example of a Bucket Brigade qRAM circuit:**
 
-### Step 1
-- Set up QDK and Python development environments
-- Get acquainted with Q#. Set up QDK and Python
-- Work through some of the quantum katas (on-going)
-- Read original bucket-brigade qRAM papers (https://arxiv.org/abs/0708.1879, https://arxiv.org/pdf/0807.4994)
+![Bucket Brigade qRAM](docs\images\bucketBrigade.gif)
+TODO: Include logo/demo screenshot etc.
 
-### Step 2
-- Read through work with basic qROM circuits (https://arxiv.org/abs/1902.01329)
-- Implement basic large-depth large-width circuits in Q#
-   - Set up framework for random-generation of such circuits (on-going; should do for these, and all future types)
+## Tech/framework used
 
-### Step 3
-- Implement bucket brigade circuits (https://arxiv.org/abs/1502.03450)
-- Implement Alexandru's constant-depth bucket-brigade circuits (https://arxiv.org/abs/2002.09340)
-- Start running circuits in the resources estimator and Toffoli simulator machines (on-going) - with random versions of these circuits, how do the gate counts compare to the ones from our paper?
+**Built with:**
 
-### Step 4
-- Implement Vadym and Guang-How's state preparation circuits (select/SWAP) (https://arxiv.org/abs/1812.00954)
-  - Get an idea of the actual constants in the runtime
-- Implement qROM from "encoding electronic spectra" paper (https://arxiv.org/abs/1805.03662)
+- [Quantum Development Kit](https://docs.microsoft.com/quantum/)
+- [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+- [Python](https://www.python.org/downloads/)
+- [Visual Studio Code](https://code.visualstudio.com/) and [Visual Studio](https://visualstudio.microsoft.com/)
+- [Jupyter Notebook](https://jupyter.org/)
 
-### Step 5
-- Writing up findings, documenting code, and getting ready for release
-    - Publish on Blog etc.
+## Features
```

Comment:
A lot of this stuff isn't filled out, is this just part of the template and we will complete it later?

Reply:
Yeah thats the idea!